recipe/meta.yaml

```diff
@@ -38,18 +38,16 @@ requirements:
     - setuptools
     - wheel
     - llvmlite 0.41.0
-    - numpy {{ numpy }}
+    - numpy 1.22.3 # [py<311]
+    - numpy {{ numpy }} # [py>=311]
     - tbb-devel 2021.8.0
     - _openmp_mutex 5.1        # [linux]
     - importlib_metadata  # [py<39]
   run:
     - python
     - importlib_metadata  # [py<39]
-    - {{ pin_compatible('llvmlite', max_pin='x.x') }}
-    # NumPy has a hard upper limit.
-    # NumPy 1.22.0, 1.22.1, 1.22.2 are all broken for ufuncs
-    # xref: https://github.com/numba/numba/issues/7756
-    - numpy >={{ numpy }},!=1.22.0,!=1.22.1,!=1.22.2,<1.26
```

Comment:
could you please explain why we remove numpy from run ?

Reply:
It's still there, two lines below:
`- {{ pin_compatible('numpy', upper_bound='1.27') }}`
Upstream changed the minimum bound of numpy to `>=1.22,<1.27,!=1.22.0,!=1.22.1,!=1.22.2`, so I updated host and run accordingly.

Reply:
does it mean we have for py 38, 39, 310 the numpy 1.22.3 in runtime ? Does not conflict what we have in global cbc.yaml ?

Reply:
The numpy pin set in the cbc is there to set a default minimum version, which defines our highest range supported (i.e. >=1.21,<2 for py38...310). At run time most users will get the highest compatible numpy version (1.26).
Here upstream, specifically set the minimum version to 1.22, so we have to follow as anyway their package is not compatible with 1.21.